src/main/java/com/conveyal/gtfs/PatternFinder.java

```diff
@@ -260,8 +265,9 @@ private static String getTerminusName(
             return stopById.get(haltId).stop_name;
         } else if (locationById.containsKey(haltId)) {
             return locationById.get(haltId).stop_name;
-        } else if (locationGroupById.containsKey(haltId)) {
-            return locationGroupById.get(haltId).location_group_name;
+        } else if (stopAreaById.containsKey(haltId)) {
+            Area area = areaById.get(haltId);
+            return area.area_name != null ? area.area_name : area.area_id;
```

Comment:
Why do we return the id as a fallback?

Reply:
@philip-cline Does the UI require the area_id? If not, can the default `fromTerminusNameUnknown` or `toTerminusNameUnknown` value be used?

Reply:
The UI doesn't require the area_id but I think it looks a lot better while we work on implementing area names.